packages/cli-repl/src/format-output.ts

```diff
@@ -298,7 +298,9 @@ export function formatError(error: Error, options: FormatOptions): string {
 
 function removeUndefinedValues<T>(obj: T) {
```

Comment:
Unrelated, but should this be:

```suggestion
function removeUndefinedValues<T>(obj: T): Partial<T> {
```

?

Reply:
![Screenshot 2023-07-11 at 19 28 23](https://github.com/mongodb-js/mongosh/assets/69737/fe07b1ef-b92f-4ac5-b15c-95e862ebaa16)

Reply:
I suppose I can fix that in turn..